src/config/entities/__tests__/configuration.ts

```diff
@@ -84,20 +84,32 @@ export default (): ReturnType<typeof configuration> => ({
       apiKey: faker.string.hexadecimal({ length: 32 }),
     },
   },
-  typeorm: { autoLoadEntities: true, manualInitialization: true },
   db: {
-    postgres: {
-      schema: process.env.POSTGRES_SCHEMA || 'main',
-      host: process.env.POSTGRES_TEST_HOST || 'localhost',
-      port: process.env.POSTGRES_TEST_PORT || '5433',
-      database: process.env.POSTGRES_TEST_DB || 'test-db',
-      username: process.env.POSTGRES_TEST_USER || 'postgres',
-      password: process.env.POSTGRES_TEST_PASSWORD || 'postgres',
-      ssl: {
-        enabled: true,
-        requestCert: true,
-        rejectUnauthorized: true,
-        caPath: process.env.POSTGRES_SSL_CA_PATH,
+    migrator: {
+      migrationsExecute: true,
+      numberOfRetries: process.env.DB_TEST_MIGRATIONS_NUMBER_OF_RETRIES ?? 5,
+      retryAfter: process.env.DB_TEST_MIGRATIONS_RETRY_AFTER ?? 1000, // Milliseconds
+    },
+    orm: {
+      autoLoadEntities: true,
+      manualInitialization: true,
+      migrationsRun: false,
```

Comment:
How does this differ from `db.migrator.migrationsExecute`?

Reply:
This is used internally by `TypeOrm`. I didn't find it necessary to add a comment about it but LMK if you want me to add a short comment in the cod explaining the difference

Reply:
As far as I understand, this achieves the same thing? If it does, I'd remove one of the two. What do you think?

Reply:
Sorry for the confusion earlier. No, it does not do the same thing. This is one of the reasons I suggest we avoid adding certain parameters to Config, as it can make things more confusing. The parameter used in orm pertains to TypeORM. If set to true, TypeORM will automatically run migrations when the module is registered, but it should always be set to false for our current use case. On the other hand, the parameter used by migrator determines whether we intend to run our custom migration logic, which includes locking the database.

Reply:
Thank you for the clarification. I now tend to agree re. package-specific logic in the configuration. I'm impartial as to _where_ it's located but I'd suggest adding some documentation as @hectorgomezv suggested.

Reply:
Done
https://github.com/safe-global/safe-client-gateway/pull/1988/commits/cfb1f8c009fd9708aef92ed5c597c8170665750f